src/libs/SearchUtils.ts

```diff
@@ -155,7 +157,7 @@ function getReportSections(data: OnyxTypes.SearchResults['data']): ReportListIte
         }
     }
 
-    return Object.values(reportIDToTransactions);
+    return Object.values(reportIDToTransactions).filter((item) => item.transactions?.length !== 0);
```

Comment:
We started filtering this on the backend, so we shouldn't have to do it in the frontend anymore.

Reply:
I put up a fix for this, so we should be able to remove the filter once my fix is deployed.